paddle/fluid/framework/tensor.cc

```diff
@@ -19,13 +19,17 @@ namespace paddle {
 namespace framework {
 extern size_t SizeOfType(proto::VarType::Type type);
 void Tensor::check_memory_size() const {
-  PADDLE_ENFORCE_NOT_NULL(
-      holder_, "Tensor holds no memory. Call Tensor::mutable_data first.");
+  PADDLE_ENFORCE_NOT_NULL(holder_, platform::errors::PreconditionNotMet(
+                                       "The `holder_` of Tensor is nullptr. "
+                                       "Tensor holds no memory. "
+                                       "Call Tensor::mutable_data first."));
   PADDLE_ENFORCE_LE(
       numel() * SizeOfType(type()), memory_size(),
-      "Tensor's dims_ is out of bound. Call Tensor::mutable_data "
-      "first to re-allocate memory.\n"
-      "or maybe the required data-type mismatches the data already stored.");
+      platform::errors::PreconditionNotMet(
+          "Tensor's dims_ is out of bound."
```

Comment:
`dims_` -> `dimension`  may be better, users may cannot understand `dims_`, we need to show users standard English word

Reply:
done, thx